grammar_checker.py

```diff
@@ -18,7 +18,7 @@ def load_dictionary(self,language = None):
             self.BIGRAMS = pickle.load(open('corpus/en_bigram.pickle', 'rb'))
             self.LETTERS = 'abcdefghijklmnopqrstuvwxyz' 
         elif language in ['ga','ga-IE']:
-            self.DICTIONARY = pickle.load(open('corpus/irish_dict.pickle', 'rb'))
```

Comment:
irish_dict.pickle is not the correct file that should be loaded, so I replaced it with irish_unigram.pickle which contains stop words.